src/script_opt/ZAM/ZBody.cc

```diff
@@ -2,6 +2,8 @@
 
 #include "zeek/script_opt/ZAM/ZBody.h"
 
+#include <type_traits>
```

Comment:
Just for my own understanding - why is this needed, given there are no other code changes?